swagger/v1/component_schemas/ParticipantDeclaration.yml

```diff
@@ -20,3 +20,4 @@ example:
       declaration_type: started
       declaration_date: 2021-05-31T02:21:32Z
       course_identifier: ecf-mentor
+      eligible_for_payment: true
```

Comment:
I think this file specifies the thing that we expect to receive from the provider. 

But `eligible_for_payment` should be in a schema that we give back to the provider. 

If we don't have a file for that already, `ParticipantDeclarationRecordedResponse`? 

Maybe we want to rename that file to `ParticipantDeclarationResponse` so we can use it in both endpoints.

Reply:
yeah, my bad, shouldn't be here

Reply:
```ParticipantDeclarationRecordedResponse```  only contains an ID as this is what's been returned from the post request

Reply:
Well, yeah, we can kill two birds with one stone and extend it. I think in the grand scheme of things we want a return after create to be in the same format as the return in get list.

But if you want you can make a new yml file for the endpoint you added in this PR, and we will merge them in a later PR.

Reply:
I've added ```ParticipantDeclarationResponse.yml``` which is referenced from ```MultipleParticipantDeclarationsResponse.yml``` as we can't reuse ```ParticipantDeclaration.yaml``` here due to new read-only flag returned by the GET participant-declaration API